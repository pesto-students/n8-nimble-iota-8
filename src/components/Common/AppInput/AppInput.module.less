.input {
    background-color: #fcfdfe;
    border-radius: 0.5em;
    border: solid 1px #f0f1f7;

    &::placeholder {
        color: #4b506d;
        opacity: 40%;
    }
}
