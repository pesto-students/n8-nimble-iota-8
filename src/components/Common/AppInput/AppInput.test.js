import React from "react";
import renderer from "react-test-renderer";
import AppInput from "src/components/Common/AppInput/AppInput";
it("ActiveMark renders correctly", () => {
    const tree = renderer.create(<AppInput />).toJSON();
    expect(tree).toMatchSnapshot();
});
