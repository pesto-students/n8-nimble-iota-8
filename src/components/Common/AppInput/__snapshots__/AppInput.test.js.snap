// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveMark renders correctly 1`] = `
<input
  className="ant-input ant-input-lg"
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  type="text"
  value=""
/>
`;
